### DISCUSSION TITLE
Caracteres chinos en Markdowns Files

### DISCUSSION THREAD

**Discussion by maikcore** [LaunchX-InnovaccionVirtual/CursoIntroPython#205] [General]
A alguien más le aparecen estos caracteres chinos en los documentos Markdown?  Alguien sabe como cambiarlos a inglés o español?

![imagen](https://user-images.githubusercontent.com/99059316/153726487-2c5ed6ab-616c-4af5-a1fa-efea4dd49c2e.png)

**Comment by DanielOe21** [comment]
Eso pasa por el editor que tienes instalado, no sé si puedes cambiarle el idioma pero en mi caso mejor lo desactivé.
